pkgs/development/python-modules/pygobject/3.nix

```diff
@@ -42,15 +42,15 @@ buildPythonPackage rec {
   ];
 
   buildInputs = [
-    # # .so files link to this
+    cairo
+    # .so files link to this
```

Comment:
The comment is true also for cairo. But perhaps the comment is redundant – why else would we add stuff `buildInputs`.